src/dal/static/autocomplete_light/forward.js

```diff
@@ -131,7 +131,7 @@
             forwardedData,
             divSelector,
             form;
-        divSelector = "div.dal-forward-conf#dal-forward-conf-for-" +
+        divSelector = "div.dal-forward-conf#dal-forward-conf-for_" +
                 element.attr("id");
```

Comment:
Looks like a pretty breaking change, any chance to support both versions at once ?

Reply:
done.

Reply:
But can you tell me about what exactly would it be breaking ? 
As I have taken care of the id in JS, Widget and Tests.